packages/actor-query-operation-source/lib/ActorQueryOperationSource.ts

```diff
@@ -44,8 +44,8 @@ export class ActorQueryOperationSource extends ActorQueryOperation {
     }
 
     const sourceWrapper: IQuerySourceWrapper = getOperationSource(action.operation)!;
-    const mergedContext = sourceWrapper.context ? action.context.merge(sourceWrapper.context) : action.context;
-
+    let mergedContext = sourceWrapper.context ? action.context.merge(sourceWrapper.context) : action.context;
+    mergedContext = setContextWrapped(action.operation, mergedContext);
```

Comment:
Are you sure this is needed here?
I would expect this context entry to only have to be set within the actor doing the actual wrapping.